poetry.lock

```diff
@@ -159,7 +159,7 @@ python-versions = "*"
 
 [[package]]
 name = "fonttools"
-version = "4.29.1"
+version = "4.31.1"
```

Comment:
This is fine in this context, but be careful about updating all dependencies when adding new ones, otherwise we get a lot of churn in dependencies, which can lead to difficulty in isolating what the cause of a "post-update-problem" is. Basically an effort to minimize parts moved.